<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>849</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PyEveLiveDPS</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow, QMenuBar, QStatusBar, QLabel, QGraphicsView { 
  background: black;
  color: white;
}
QStatusBar::item { border: none; }
QMenuBar::item:selected { background: rgb(60, 60, 60) }
QMessageBox QLabel {
  color: black;
  background: white;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
  background: black;
  color: white;
  padding:2px 0px 2px 0px;
  border-style: none;
}
QPushButton:hover { background: rgb(60, 60, 60) }
QPushButton:pressed { background: rgb(85, 85, 85) }
</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="mainGrid">
      <item row="7" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>GRAPH</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="topGrid" columnstretch="0,0,0,0,0" columnminimumwidth="0,0,0,0,0">
        <property name="horizontalSpacing">
         <number>3</number>
        </property>
        <item row="0" column="1">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="4">
         <widget class="QPushButton" name="buttonQuit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="buttonCollapse">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="buttonMinimize">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <spacer name="menuBarLocation">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="6" column="0">
       <layout class="QGridLayout" name="labelGrid">
        <item row="1" column="1">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>DPS Out</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>DPS In</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>849</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="mouseTracking">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File...</string>
    </property>
    <widget class="QMenu" name="menuProfiles">
     <property name="title">
      <string>Profiles</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionAdd_New_Profile"/>
     <addaction name="actionDuplicate_Current_Profile"/>
     <addaction name="actionRename_Current_Profile"/>
     <addaction name="actionDelete_Current_Profile"/>
    </widget>
    <addaction name="actionEditSettings"/>
    <addaction name="menuProfiles"/>
    <addaction name="separator"/>
    <addaction name="actionFleetMode"/>
    <addaction name="separator"/>
    <addaction name="actionSimulateInput"/>
    <addaction name="actionPlaybackLog"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuCharacter">
    <property name="title">
     <string>Character...</string>
    </property>
    <addaction name="actionNoCharacters"/>
    <addaction name="separator"/>
    <addaction name="actionOverviewSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuCharacter"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOverviewSettings">
   <property name="text">
    <string>Open Overview Settings</string>
   </property>
  </action>
  <action name="actionNo_character_logs_detected_for_past_24_hours">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No character logs detected for past 24 hours</string>
   </property>
  </action>
  <action name="actionNoCharacters">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>No character logs detected for past 24 hours</string>
   </property>
  </action>
  <action name="actionEditSettings">
   <property name="text">
    <string>Edit Profile Settings</string>
   </property>
  </action>
  <action name="actionAdd_New_Profile">
   <property name="text">
    <string>Add New Profile</string>
   </property>
  </action>
  <action name="actionDuplicate_Current_Profile">
   <property name="text">
    <string>Duplicate Current Profile</string>
   </property>
  </action>
  <action name="actionRename_Current_Profile">
   <property name="text">
    <string>Rename Current Profile</string>
   </property>
  </action>
  <action name="actionDelete_Current_Profile">
   <property name="text">
    <string>Delete Current Profile</string>
   </property>
  </action>
  <action name="actionFleetMode">
   <property name="text">
    <string>Fleet Mode</string>
   </property>
  </action>
  <action name="actionSimulateInput">
   <property name="text">
    <string>Simulate Input</string>
   </property>
  </action>
  <action name="actionPlaybackLog">
   <property name="text">
    <string>Playback Log</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
